docs/install/network_options.md

```diff
@@ -138,6 +138,14 @@ Each CNI plugin requires a different configuration for dual-stack:
     Calico automatically detects the RKE2 configuration for dual-stack and does not need any extra configuration. When deployed in dual-stack mode, it creates two different ippool resources. Note that when using dual-stack, calico leverages BGP instead of VXLAN encapsulation. Dual-stack and BGP are currently not supported in the windows installations of RKE2.
 
 
+## IPv6 setup
+
+In case of IPv6 only configuration RKE2 needs to use `localhost` to acces the liveness URL of the ETCD pod; it needs to be rightly configured on `/etc/hosts` file.
```

Comment:
```suggestion
In case of IPv6 only configuration RKE2 needs to use `localhost` to acces the liveness URL of the ETCD pod; check that your operating system configures `/etc/hosts` file correctly:
```